mapfishapp/src/main/webapp/app/js/GEOR_wmcbrowser.js

```diff
@@ -418,6 +419,22 @@ GEOR.wmcbrowser = (function() {
             }, view, formPanel]
         });
     };
+    
+    var filterContexts = function(storeData) {
+		for(var i = storeData.length; i--;){
+			var remove = true;
+			if( storeData[i].roles == undefined || storeData[i].roles.length === 0 ) continue;
+			for (var j = 0; j < storeData[i].roles.length; j++) {
+				if (GEOR.config.ROLES.indexOf(storeData[i].roles[j]) >= 0) {
+					remove = false;
+					break;
+				}
+			}
+			if(remove) storeData.splice(i, 1);
```

Comment:
When I do a single command after if I don't put parenthesis, but it's a coding method, ;)

I added them